deploy-acm/deploy.sh

```diff
@@ -21,38 +21,31 @@ if ./verify.sh; then
 
     echo ">>>> Deploy manifests to install ACM ${OC_ACM_VERSION}"
     echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
-    oc apply -f 01-namespace.yml -o yaml --dry-run=client | oc apply -f -
+    oc apply -f 01-namespace.yml
     sleep 2
     oc apply -f 02-operatorgroup.yml
     sleep 2
     oc apply -f 03-subscription.yml
-    sleep 240
-
-    echo ">>>> Deploy ACM cr manifest"
-    echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    sleep 40
+    InstallPlan=$(oc --kubeconfig=${KUBECONFIG_HUB} get installplan -n open-cluster-management -o name)
+    RESOURCE_KIND=${InstallPlan%%/*}
+    RESOURCE_NAME=${InstallPlan##*/}
+    check_resource "${RESOURCE_KIND}" "${RESOURCE_NAME}" "Deployed" "open-cluster-management" "${KUBECONFIG_HUB}"
+
```

Comment:
`oc  -n ${NAMESPACE} get ${RESOURCE} ${RESOURCE_NAME} -o jsonpath="{.status.conditions}"               `


Gives:

`[{"lastTransitionTime":"2022-05-25T09:41:01Z","lastUpdateTime":"2022-05-25T09:41:01Z","status":"True","type":"Installed"}]`

Note the removed 'TYPE_STATUS'... so it keeps in the loop as it cannot verify the status

Reply:
![image](https://user-images.githubusercontent.com/312463/170242130-9eba9cc1-4601-4dab-824e-698c770a96ff.png)